web-ui/view/github.ml

```diff
@@ -27,6 +27,21 @@ let github_pr_url ~org ~repo id =
 
 let format_org org = li [ a ~a:[ a_href (org_url org) ] [ txt org ] ]
 
+let ref_name r = 
+  match Astring.String.cuts ~sep:"/" r with
+  | "refs" :: "heads" :: branch -> Astring.String.concat ~sep:"/" branch
+  | [ "refs"; "pull"; id; "head" ] -> id
+  | _ -> Fmt.str "Bad ref format %S" r
+
+let ref_breadcrumb r head_hash =
+  match Astring.String.cuts ~sep:"/" r with
+  | "refs" :: "heads" :: branch ->
+      let branch = Astring.String.concat ~sep:"/" branch in
+      (branch, Fmt.str "commit/%s" head_hash)
```

Comment:
You don't need `Fmt` here, use `Printf.sprintf` (or an alias). `Fmt` has the full power of the formatting engine but you're just creating simple strings with simple formats.